hetmech/test_degree_weight.py

```diff
@@ -0,0 +1,31 @@
+import hetio.readwrite
+import pytest
+
+from .degree_weight import dwwc
+from .matrix import get_node_to_position
+
+
+def test_disease_gene_example_dwwc():
+    """
+    Test the PC & DWWC computations in Figure 2D of Himmelstein & Baranzini
+    (2015) PLOS Comp Bio. https://doi.org/10.1371/journal.pcbi.1004259.g002
+    """
+    url = 'https://github.com/dhimmel/hetio/raw/{}/{}'.format(
+        '9dc747b8fc4e23ef3437829ffde4d047f2e1bdde',
+        'test/data/disease-gene-example-graph.json',
+    )
+    graph = hetio.readwrite.read_graph(url)
+    metagraph = graph.metagraph
+
+    # Compute GiGaD path count and DWPC matrices
+    metapath = metagraph.metapath_from_abbrev('GiGaD')
+    pc_matrix = dwwc(graph, metapath, damping=0)
+    dwwc_matrix = dwwc(graph, metapath, damping=0.5)
+
```

Comment:
I just want to clarify that in *this* case `dwwc( graph, metapath, damping=0)` produces the path count, but only because the requested metapath GiGaD does not repeat nodes.

In general this could would return the walk count.